### ISSUE TITLE
OWL-ViT inference playround: module 'bokeh.plotting' has no attribute 'Figure'

### ISSUE THREAD

**Issue by jacky-ch93** [google-research/scenic#859]
Get the Error when using `OWL-ViT inference playround.ipynb` when run the code below
```python
import os

from bokeh import io as bokeh_io
import jax
from google.colab import output as colab_output 
import matplotlib as mpl
from matplotlib import pyplot as plt
import numpy as np
from scenic.projects.owl_vit import models
from scenic.projects.owl_vit.configs import clip_b16 as config_module
from scenic.projects.owl_vit.notebooks import inference
from scenic.projects.owl_vit.notebooks import interactive
from scenic.projects.owl_vit.notebooks import plotting
from scipy.special import expit as sigmoid
import skimage
from skimage import io as skimage_io
from skimage import transform as skimage_transform
import tensorflow as tf

tf.config.experimental.set_visible_devices([], 'GPU')
bokeh_io.output_notebook(hide_banner=True)
```
error is as below showing
```python
AttributeError                            Traceback (most recent call last)
[<ipython-input-2-6cf721b232cb>](https://localhost:8080/#) in <cell line: 13>()
     11 from scenic.projects.owl_vit.notebooks import inference
     12 from scenic.projects.owl_vit.notebooks import interactive
---> 13 from scenic.projects.owl_vit.notebooks import plotting
     14 from scipy.special import expit as sigmoid
     15 import skimage

[/content/scenic/projects/owl_vit/notebooks/plotting.py](https://localhost:8080/#) in <module>
    118                          fig_size: int = 900,
    119                          title: str = '',
--> 120                          tools: str = '') -> plotting.Figure:
    121   """Creates a Bokeh figure showing an image."""
    122   # Determine relative width and height from padding. We assume that padding is

AttributeError: module 'bokeh.plotting' has no attribute 'Figure'
```

**Comment by merlinxu-zz** [comment]
try to replace 'Figure' with 'figure', it works for me

**Comment by mjlm** [comment]
Thanks for reporting, this will be fixed in #892. Please re-open if the issue persists.
